CONTRIBUTING.md

```diff
@@ -11,25 +11,31 @@ Most of the time, if webpack is not working correctly for you it is a simple con
 
 If you are still having difficulty after looking over your configuration carefully, please post
 a question to [StackOverflow with the webpack tag](http://stackoverflow.com/tags/webpack). Questions
-that include your webpack.config.js and relevant files are more likely to receive responses.
+that include your webpack.config.js, relevant files, and the full error message are more likely to receive responses.
 
 **If you have discovered a bug or have a feature suggestion, please [create an issue on GitHub](https://github.com/webpack/webpack/issues/new).**
 
+Do you want to fix an issue?  Look at the issues with a tag of [X5: work required (PR / Help Wanted)](https://github.com/webpack/webpack/labels/X5%3A%20work%20required%20%28PR%20%2F%20Help%20Wanted%29).  Each issue should be tagged with a difficulty tag -
+
+- D0: My First Commit (Contrib. Difficulty)
```

Comment:
Could you please expand `Contrib.`?

Reply:
Yes, @montogeek is it contribute or contributing?

Reply:
Contribution I think

Reply:
Updated.  Let me know if it looks good.